apps/frontend/src/types/domain.ts

```diff
@@ -0,0 +1,145 @@
+// Domain types that mirror the backend exactly
+// Keep in sync with apps/backend/src/repositories/*
```

Comment:
Can we share the types? One of things we both thought we could improve from the PoC was to share types between the frontend and the backend. Let's do that instead of copying and leaving comments to keep things up to date!